docs/upgrade_with_cli.md

```diff
@@ -0,0 +1,60 @@
+# Upgrade Percona Everest using the CLI
+
+Percona Everest regularly releases updates that contain bug fixes, security improvements, and other enhancements to improve the overall performance of your database.
+
+
+## Before you upgrade
+
+!!! note
+     It is recommended to install the latest version of the CLI .
+
+You need to download CLI version >=0.10.0 for the upgrade command to work. You can only upgrade to one version at a time. For instance, you can upgrade from version 0.9.1 to version 0.10.0, but not directly from version 0.8 to version 0.10.  
+
+## Upgrade
+
+!!! note
+    It is recommended that you create backups of your databases prior to upgrading Percona Everest.
+
+During the upgrade of Percona Everest, only Everest and Everest operator are upgraded, whereas the database operators, database clusters and backups remain unchanged.
+{.power-number}
+
+1. To upgrade Percona Everest using the CLI, execute the following command:
+
+    everestctl upgrade
```

Comment:
Not formatted correctly

Reply:
Fixed

Reply:
Did you forget to push the commit? I still see the same...

Reply:
it's correct now